README.rst

```diff
@@ -87,4 +87,113 @@ the exporting file and override ``get_sharing_configs_export_data`` method
     :target: https://pypi.org/project/sharing_configs/
 
 .. _Maykin Media B.V.: https://www.maykinmedia.nl
-.. _Sharing Configs API: https://github.com/maykinmedia/sharing-configs-api.git
\ No newline at end of file
+.. _Sharing Configs API: https://github.com/maykinmedia/sharing-configs-api.git
+
+Example
+=======
+Simple example for changing color theme in the django admin:
```

Comment:
The example should focus on the sharing aspect. Now, the example focuses on the color switching. So, let's rewrite it a little bit:

```suggestion

We can use the Sharing Configs library to exchange color-themes for the Django admin with other users. For this, we need a model that stores the color-theme, and use the Sharing Configs mixins to import and export the color-theme.
```